bundles/framework/personaldata/PublishedMapsTab.js

```diff
@@ -353,8 +401,24 @@ Oskari.clazz.define('Oskari.mapframework.bundle.personaldata.PublishedMapsTab',
             // set up the link from name field
             var showRenderer = function (name, data) {
                 var link = me.templateLink.clone();
+                var srs = data.state.mapfull.state.srs;
                 link.text(name);
                 link.bind('click', function () {
+
+                    var cb = function (url) {
+                        window.open(
+                            url(),
+                            'Published',
+                            'location=1,status=1,scrollbars=yes,width=850,height=800'
+                        );
+                    }
+
+                    var supported = me._isCurrentProjectionSupportedForView( data );
+                    if (!supported) {
+                        me.createProjectionChangeDialog( cb.bind(me, me.constructUrlUuid.bind(this, srs) ));
```

Comment:
What does this do? Seems kinda complex solution with lots of binds where it could be done more simply.

Reply:
Could be written as:

    me.createProjectionChangeDialog( function() {
	    window.open(
            me.constructUrlUuid(srs),
            'Published',
            'location=1,status=1,scrollbars=yes,width=850,height=800'
         );
    });

Which I think is much more readable and easier to understand.